src/Illuminate/Foundation/Testing/Concerns/MocksApplicationServices.php

```diff
@@ -88,7 +89,14 @@ protected function withoutEvents()
             $this->firedEvents[] = $called;
         });
 
+        $mock->shouldReceive('until')->andReturnUsing(function ($called) {
+            $this->firedEvents[] = $called;
+
+            return true;
+        });
+
         $this->app->instance('events', $mock);
+        Model::setEventDispatcher($mock);
```

Comment:
_maybe_? The application is rebooted on every test, right? So wouldn't that reset the event dispatcher by running the service providers?